tests/test_get_read_orientation.py

```diff
@@ -1,5 +1,8 @@
 """Unit tests for module ``get_read_orientation.py``."""
```

Comment:
Commenting here because I can't comment on an empty file and I don't want to comment on a 170'000 lines file :)

* It doesn't really make sense to add a description and file extension to an empty file. Just call it `empty` and the variable `FILE_EMPTY`, it's shorter, more precise and makes more sense to reuse in other contexts.
* Why add `transcripts.fasta` (a big file!) again, when a compressed version is already in `data/transcripts.fasta.gz`? Couldn't you just use that? If not, for whatever reason, I think it would make sense to just use a small (max 10'000 lines, better 1'000) subset of the file.

Reply:
Okay, probably better to have a subset in the test files. We already have the complete file in `data/`

Reply:
Well, my issue is that there is a big (10 Mb) uncompressed file in `tests/files`, and this file was added in _this_ PR. And I don't think it's (a) necessary that we have such a big test file and (b) a compressed version of the file is _already_ in the repo (in `data/`). Why not use the file in `/data`?

And you didn't just rename the file. 170+k lines were added, 3+k lines were removed:
![image](https://user-images.githubusercontent.com/10855418/194779559-adf0a7e3-8824-4200-91b1-40072a5011af.png)

As for subsetting, you don't need to open the file. Just use a tool like `grep` to subset.